Formula/xcodegen.rb

```diff
@@ -0,0 +1,28 @@
+require "English"
+
+class Xcodegen < Formula
+  desc "Tool that generates your Xcode project from a project spec."
+  homepage "https://github.com/yonaskolb/XcodeGen"
+  url "https://github.com/yonaskolb/XcodeGen/archive/#{version}.tar.gz"
```

Comment:
Please include the version directly in the URL, and let Homebrew auto-detect it if possible